docs/core/testing/mstest-analyzers/mstest0026.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "MSTEST0026: Avoid conditional access in assertions"
+description: "Learn about code analysis rule MSTEST0026: Avoid conditional access in assertions"
+ms.date: 05/15/2024
+f1_keywords:
+- MSTEST0026
+- AssertionArgsShouldAvoidConditionalAccessRuleId
+helpviewer_keywords:
+- AssertionArgsShouldAvoidConditionalAccessRuleId
+- MSTEST0026
+author: fhnaseer
+ms.author: faisalhafeez
+---
+# MSTEST0026: Avoid conditional access in assertions
+
+| Property                            | Value                                                 |
+|-------------------------------------|-------------------------------------------------------|
+| **Rule ID**                         | MSTEST0026                                            |
+| **Title**                           | Avoid conditional access in assertions                |
+| **Category**                        | Usage                                                 |
+| **Fix is breaking or non-breaking** | Non-breaking                                          |
+| **Enabled by default**              | Yes                                                   |
+| **Default severity**                | Info                                                  |
+| **Introduced in version**           | 3.5.0                                                 |
+
+## Cause
+
+This rule raises a diagnostic when an argument containing a conditional access `(?.)` is passed to the assertion methods below:
```

Comment:
We are also handling conditional indexer :)

Reply:
Added.